.github/workflows/dusk_ci.yml

```diff
@@ -15,6 +15,14 @@ jobs:
     name: Dusk Analyzer
     uses: dusk-network/.github/.github/workflows/dusk-analysis.yml@main
 
+  build_benches:
+    name: Build Benchmarks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
You can use checkout@v4 here, uses a newer version of Node.js.

Reply:
I copied from plonk so we might want to update this there as well